assets/src/edit-story/output/story.js

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import StoryPropTypes from '../types';
+import { OutputPage } from './index';
+
+// todo: improve / move elsehwere.
+const getUsedAmpExtensions = (pages) => {
+  const extensions = [
+    // runtime.
+    { src: 'https://cdn.ampproject.org/v0.js' },
+    {
+      name: 'amp-story',
+      src: 'https://cdn.ampproject.org/v0/amp-story-1.0.js',
+    },
+  ];
+
+  for (const { elements } of pages) {
+    for (const { type } of elements) {
+      switch (type) {
+        // Todo: eventually check for amp-fit-text if ever added.
```

Comment:
Do we need `amp-font` here?

Reply:
We do not use or rely on `amp-font` anywhere.